### ISSUE TITLE
Too large `step_size` in deepedit

### ISSUE THREAD

**Issue by KumoLiu** [Project-MONAI/tutorials#1430]
**Describe the bug**
Too large `step_size` which may cause the learning rate never decreases unless the model is trained for 5000 epochs.
https://github.com/Project-MONAI/tutorials/blob/bbc4e180f5130859396a98e35523bc73fa694595/deepedit/ignite/train.py#L268

**Expected behavior**
Set smaller `step_size` or set `epoch_level=False` in `LrScheduleHandler`

Originally posted in https://github.com/Project-MONAI/MONAI/issues/6625

**Comment by KumoLiu** [comment]
Hi @diazandr3s, could you please help take a look at this issue?
Thanks in advance!

**Comment by KumoLiu** [comment]
Hi @diazandr3s, I didn't try to reduce the `step_size`, I'm assuming there's a typo here because the value of 5000 is set too high if it's epoch-level.

Thanks!
